tests/installation/ipxe_install.pm

```diff
@@ -15,7 +15,7 @@ use testapi;
 use bmwqemu;
 use ipmi_backend_utils;
 use version_utils qw(is_upgrade is_tumbleweed);
-use bootloader_setup 'prepare_disks';
+use bootloader_setup qw(prepare_disks bootmenu_default_params specific_bootmenu_params);
```

Comment:
Shall we think about remove `bootmenu_default_params` `specific_bootmenu_params` in `use`, and do not touch `bootloader_setup.pm`?

Reply:
I am managing to append these arguments directly.